### PR TITLE
chore(TBD-12114):Remove Storm code generator

### PR DESCRIPTION
**Pull request by lbourgeois**
- wip
- wip

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.codegen/src/main/java/org/talend/designer/codegen/config/NodesSubTree.java [hunk 1]
```diff
@@ -121,16 +121,6 @@ public NodesSubTree(INode node, List<? extends INode> nodes, ETypeGen typeGen) {
             allMainSubTreeConnections = new ArrayList<IConnection>();
 
             buildCamelSubTree(node, false);
-        } else if ((typeGen == ETypeGen.MR) || (typeGen == ETypeGen.STORM)) {
```
Comment (lbourgeois) [inline main/plugins/org.talend.designer.codegen/src/main/java/org/talend/designer/codegen/config/NodesSubTree.java:124]: wrong removal

Comment (lbourgeois) [reply main/plugins/org.talend.designer.codegen/src/main/java/org/talend/designer/codegen/config/NodesSubTree.java:124]: Shouldn't we keep else if ((typeGen == ETypeGen.MR) ?

**Review by cbadillo1603 — APPROVED** [review]
reviewed

**Comment by nrousseau** [comment]
It's a feature for 8.0.. why there is even a PR for 7.3?

**Pull request closed without merging**